onboardProject/utils/apollo.tsx

```diff
@@ -0,0 +1,26 @@
+import { ApolloClient, HttpLink, InMemoryCache } from "@apollo/client";
+import { AsyncStorage } from "react-native";
+
+export const client = new ApolloClient({
+    cache: new InMemoryCache(),
+    link: new HttpLink({
+        uri: "https://tq-template-server-sample.herokuapp.com/graphql"
+    })
+});
+
+export const _storeData = async (key:string, value:string) => {
```

Comment:
Por que o `_` antes?

Reply:
Padrão python para especificação de conteúdo privado. Posso tirar se for o caso de não ser algo válido

Reply:
No caso do javascript, ja vi pessoas usando para métodos privados de classe. Mas nesse caso esse método é do módulo `apollo`, e está exportado. De qualquer forma, como você consegue exportar apenas o que será usado fora, não vejo muita necessidade do `_`